library/core/src/ffi/c_str.rs

```diff
@@ -513,6 +489,30 @@ impl CStr {
         self.inner.as_ptr()
     }
 
+    /// Returns the length of `self`.
```

Comment:
Should this mention that it's the number of bytes *not including* the terminator, for clarity?

Reply:
Good thought, I will add this

Reply:
Added this in my most recent commit